index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="main-content">
+        <div class="topbar">
+            <button type="button" class="topbar-button"><i class="fab fa-apple"></i></button>
+            <button type="button" class="topbar-button">Store</button>
+            <button type="button" class="topbar-button">Mac</button>
+            <button type="button" class="topbar-button">iPad</button>
+            <button type="button" class="topbar-button">iPhone</button>
+            <button type="button" class="topbar-button">iTunes</button>
+            <button type="button" class="topbar-button">Support</button>
+            <form action="#" class="topbar-search">
+                <input type="search" placeholder="Search">
+            </form>
+        </div>
+        <div class="header">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the
+                world’s story.</h2>
+            <h3>Watch the “Your Verse” film and explore the stories</h3>
+        </div>
+    </div>
+    <div class="footer-grid">
+        <div class="footer-element">
```

Comment:
Why did you decide to go for a gradient and not an image?

Reply:
That's in the project name :) We follow through the guidelines. Also, we wanted to show our learnings about it.